### PR TITLE
Example centos walkthrough, add working secure apache config

### PR DESCRIPTION
**Pull request by AndrewwHummer**
It would be really helpful if the example of a CentOS Configuration Walkthrough would directly contain a working and secure apache configuration.
Just the Reference to the detailed apache config is not adequate and too easy to overlook and users will end up with an unsecure and broken config. This could lead to the usual htaccess not working questions.

So here is my proposal :)

### CODE REVIEW

File: admin_manual/installation/example_centos.rst [hunk 1]
```diff
@@ -21,8 +21,26 @@ Apache
...
+    <Directory /var/www/html/nextcloud/>
+      Require all granted
+      AllowOverride All
+      Options FollowSymLinks MultiViews
+
+      <IfModule mod_dav.c>
+        Dav off
+      </IfModule>
 
-See :ref:`apache_configuration_label` for details.
```
Comment (MorrisJobke) [inline admin_manual/installation/example_centos.rst:25]: I would keep this to provide further documentation as well.

Comment (AndrewwHummer) [reply admin_manual/installation/example_centos.rst:25]: good point

**Comment by AndrewwHummer** [comment]
Is something still missing, or can me merge this ? :)

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**